video/out/opengl/hwdec_d3d11egl.c

```diff
@@ -100,12 +100,14 @@ static int init(struct ra_hwdec *hw)
 
     GL *gl = ra_gl_get(hw->ra);
 
+    const char *client_exts = eglQueryString(EGL_NO_DISPLAY, EGL_EXTENSIONS);
     const char *exts = eglQueryString(egl_display, EGL_EXTENSIONS);
     if (!gl_check_extension(exts, "EGL_ANGLE_d3d_share_handle_client_buffer") ||
         !gl_check_extension(exts, "EGL_ANGLE_stream_producer_d3d_texture") ||
         !(gl_check_extension(gl->extensions, "GL_OES_EGL_image_external_essl3") ||
           gl->es == 200) ||
-        !gl_check_extension(exts, "EGL_EXT_device_query") ||
+        !(gl_check_extension(client_exts, "EGL_EXT_device_query") ||
+          gl_check_extension(exts, "EGL_EXT_device_query")) ||
```

Comment:
if its a client extension why keep the second check?

Reply:
If I understand the comments in #10212 correctly, before November 2020 ANGLE exposed this in the old location, while current binaries like the one I just tested this with (https://github.com/google/gfbuild-angle/releases/tag/github%2Fgoogle%2Fgfbuild-angle%2Ff810e998993290f049bbdad4fae975e4867100ad) expose it in the new location.

This PR seems to attempt to support both ages of libraries.

Reply:
supporting old version is trivial work. some users may use outdated angle provided by framework(eg. qt?)